python/tvm/micro/rpc_server.py

```diff
@@ -0,0 +1,90 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""RPC server for interacting with devices via MicroTVM"""
+
+import argparse
+import ast
+import ctypes
+import json
+import logging
+import os
+import tvm
+from tvm import rpc
+from tvm import micro
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--host', type=str, default="0.0.0.0",
+                        help='the hostname of the server')
+    parser.add_argument('--port', type=int, default=9091,
+                        help='The port of the RPC')
+    parser.add_argument('--port-end', type=int, default=9199,
+                        help='The end search port of the RPC')
+    parser.add_argument('--key', type=str, default="",
+                        help='RPC key used to identify the connection type.')
+    parser.add_argument('--tracker', type=str, default="",
+                        help='Report to RPC tracker')
```

Comment:
can we patch tvm.exec.rpc_server to include ```utvm-dev-config``` options to be utvm compatible?

Reply:
sure. though we'll also need to add [these overrides](https://github.com/apache/incubator-tvm/pull/4274/files/ebaac5d4d790d9f8edbb16328a744e1220e6403b#diff-c7f1f501188eb16092163484ecbe2327R72-R79), if that's okay

Reply:
@tqchen long term, do we also want to merge the [VTA RPC server](https://github.com/apache/incubator-tvm/blob/master/vta/python/vta/exec/rpc_server.py) with `tvm.exec.rpc_server`?